deployment/docker/snmp-base-nr.yaml

```diff
@@ -0,0 +1,27 @@
+# Recommended default minimalist config for sending data to New Relic
+---
+devices:
+trap:
+  listen: 127.0.0.1:1620
+discovery:
+  cidrs:
+  - 10.10.0.0/24
```

Comment:
Is there a reason we default to `10.10.0.0/24`? 

I mentioned this before and think it's worth bringing up again: would it be useful to have ktranslate use `<host_ip>/24` as a default, and use `{P,p}ublic / {P,p}rivate` as the community strings? Reason being - this might be a sensible default to just get a user going quickly and it'd discover reachable devices attached to that host. It wouldn't be the true production deployment, but at that point users are likely getting more involved in the configuration anyways. We could still have input fields in the UI so the user can edit CIDRs and community strings, but if they omit then we can fall back to these defaults.

We'd just have to figure out some way to indicate that behavior in the config. I floated this idea by @i3149 a few weeks ago and he indicated it'd be feasible and might make sense as a sensible default. Thoughts?

Reply:
I'll throw in that as a general rule I have been strongly discouraging anyone from using ktrans to monitor servers because net-snmp is such a huge downgrade compared to our agent.   Making it the default entity undermines that a bit, but I think the value in just being able to discover a single local thing without relying on other teams for firewalls and access and such is worth it.

Reply:
`10.10.0.0/24` Is how my home network is set up. No better reason that this for why the default.

Reply:
> but I think the value in just being able to discover a single local thing without relying on other teams for firewalls and access and such is worth it.

I think there's a discussion to be had around this. IMO there should be a purposeful drive to installation of this agent and opening up paths towards monitoring entities following anti-patterns just to show success with ktranslate seems incorrect. I'm trying to think of the value as a user of seeing ktranslate able to monitor a server via SNMP vs a better experience that guides me towards getting my firewall/access fixed.

Reply:
Agreed, a discussion would be good. I don't think we'd want to drive monitoring of a server via SNMP (we'd push the infra agent for that use case). I was more so wondering if it's feasible to reach even the first network hop with some sensible defaults. Get that first device into NR and then figure out how to get more. But if it's not worth the time/effort + will be error prone, then we can just ignore my musings and go back to regularly scheduled programming 😄